### PR TITLE
Clearer instructions for example ros1->ros2

### PR DESCRIPTION
**Pull request by jlblancoc**
Related to #205. I also found the instructions unclear about which ROS versions one has to source to run the bridge, so the proposal is to make it clearer.

### CODE REVIEW

**Comment by jlblancoc** [comment]
Agreed DCO after opening the PR... probably you can re-run the check? if not, feel free of closing and re-opening a new one. Cheers

File: README.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ You will get errors from most tools if they have both workspaces in their enviro
 First we start a ROS 1 `roscore`:
 
 ```
-# Shell A:
+# Shell A (ROS1 only):
```
Comment (dirk-thomas) [inline README.md]: Please use a space between `ROS` and the major version number.

Same below multiple times.

**Comment by dirk-thomas** [comment]
> Agreed DCO after opening the PR... probably you can re-run the check?

Actually you have to update the commit message to contain the line `Signed-off-by: ...` (see https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#pull-requests). You can fix the existing commit with `git commit --amend -s` and then force push that change to your branch: `git push -f`. The DCO check will pass.

**Comment by jlblancoc** [comment]
Oh, sure! Didn't quite understood the purpose of the DCO app.
Done now. Thanks @dirk-thomas .

**Comment by dirk-thomas** [comment]
Thanks for the improvement.

**Pull request merged**